README.adoc

```diff
@@ -143,7 +143,7 @@ This module provides many endpoints, including the `HealthEndpoint`, `Environmen
 This provides auto-configuration for actuator endpoints based on the content of the classpath and a set of properties.
 For instance, if Micrometer is on the classpath, it will auto-configure the `MetricsEndpoint`.
 It contains configuration to expose endpoints over HTTP or JMX.
-Just like Spring Boot AutoConfigure, this will back away as the user starts to define their own beans.
```

Comment:
I'm not sure why own is being removed here.

Reply:
"Own" is redundant in this case as we are already saying user   _starts to define their beans_, so there is no need for "own" according to me. Rest the decision is up to you.